docs/conf.py

```diff
@@ -8,18 +8,19 @@
 import sys
 
 
-sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
-
-
 # -- Project information -----------------------------------------------------
 
 project = "unidist"
 copyright = "2021-2022, Modin Authors"
 author = "unidist contributors"
 
-# The full version, including alpha/beta/rc tags
-release = "0.0.1"
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), "..")))
+import unidist  # noqa: E402
 
+# The short project version like 1.1
+version = f"{unidist.__version__}"
+# The full project version like 1.1.0rc1
+release = version
```

Comment:
I didn't quite get it. Why version and release are equal?

Reply:
This is for sphinx settings. Each var is responsible for different concepts.